osquery/config/tests/packs_tests.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#include <gtest/gtest.h>
+
+#include <osquery/config/packs.h>
+#include <osquery/core.h>
+#include <osquery/filesystem.h>
+
+#include "osquery/core/test_util.h"
+
+namespace osquery {
+
+class PacksTests : public testing::Test {};
+
+pt::ptree getExamplePacksConfig() {
+  std::string content;
+  auto s = readFile(kTestDataPath + "test_inline_pack.conf", content);
+  assert(s.ok());
+  std::stringstream json;
+  json << content;
+  pt::ptree tree;
+  pt::read_json(json, tree);
+  return tree;
+}
+
+pt::ptree getKernelPack() {
+  auto tree = getExamplePacksConfig();
```

Comment:
You could remove some of this duplicate code with an inline or `json` yielding method.